kernel/configs/x86_64_kata_kvm_4.19.x

```diff
@@ -3016,6 +3017,11 @@ CONFIG_STACK_VALIDATION=y
 # CONFIG_MAGIC_SYSRQ is not set
 CONFIG_DEBUG_KERNEL=y
 
+#
+# Fuzzing
+#
+CONFIG_KCOV=n
```

Comment:
This looks like a testing feature. Why is this required @GabyCT ?

Reply:
@amshinde when you install the kernel script using the `https://github.com/kata-containers/tests/blob/master/.ci/install_kata_kernel.sh`, it will not run completely as it is asking about this config options. That is the main reason behind this.